src/test/java/com/solvd/laba/LoginUserTest.java

```diff
@@ -0,0 +1,44 @@
+package com.solvd.laba;
+
+import java.util.Map;
+
+import com.solvd.laba.config.TestDataConfig;
+import com.solvd.laba.pages.LoginPage;
+import com.solvd.laba.utils.UserFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginUserTest extends BaseTest{
+
+    @Test
+    public void testLoginWithCorrectUserCredentials(){
+        openHomePage();
+
+        logger.info("➡️ Open Login Page");
+        LoginPage loginPage = homePage.header().clickSignupLogin();
+        Assert.assertTrue(loginPage.isOpened(), "Login page should be opened");
+
+        logger.info("🔐Login with credentials from testdata.properties");
+        loginPage.loginOnSite(email, password);
+
+        logger.info("✅ Verify login was successful");
+        Assert.assertTrue(loginPage.header().isUserLoggedIn(testData.getProperty("name")));
+        Assert.assertTrue(homePage.isOpened(), "Home page should be opened after login");
+    }
+
+    @Test
+    public void testLoginWithIncorrectUserCredentials(){
```

Comment:
To reduce code duplication and utilize a DataProvider to run a single test with different sets of data, you can combine both tests into one and use the @DataProvider annotation